### DISCUSSION TITLE
Weird beeping segments

### DISCUSSION THREAD

**Discussion by Dk5694** [bitcraze/discussions#701] [Q&A]
I have recently tried to fix my first quadcopter. I have a crazyflie with the flowdeck v2, big quad deck, a board to step down the voltage from 12V to 5V, and an iFlight Blitz E55 32Bit 55A 2-6S 30x30 4in1 ESC. It seems as though the recently updated crazyflie and the ESC aren't communicating properly. I may need to flash the ESC firmware but that's just me making a guess. If it has already been posted I apologize, I'm not even sure how to search this. Here is a video https://www.dropbox.com/s/5skggdze4hrz7p8/IMG_7739.MOV?dl=0 and https://www.dropbox.com/s/l6xkc1pp3xutug5/IMG_7741.MOV?dl=0

**Comment by tobbeanton** [comment]
Hi!

My guess is that the ESC signal is not correctly detected. Have you powered the BigQuad deck with 5V? This is required for the ESC signal to reach the correct level.

The ESC is running BLHeli32 and the manual (with beep codes) can be found [here](https://github.com/bitdump/BLHeli/blob/master/BLHeli_32%20ARM/BLHeli_32%20manual%20ARM%20Rev32.x.pdf).
